.circleci/workflows.yml

```diff
@@ -1910,7 +1906,7 @@ jobs:
       - restore_cached_workspace
       - run:
           name: Build
-          command: yarn lerna run build --scope @cypress/webpack-preprocessor
+          command: yarn build --scope @cypress/webpack-preprocessor
```

Comment:
No need to reference Lerna specifically in these scripts, just call `yarn build` and pass the arguments through.